contents/blog/best-eppo-alternatives.md

```diff
@@ -0,0 +1,452 @@
+
+- 💬 **Surveys:** Target surveys by event or person properties. Templates for [net promoter score (NPS)](/templates/nps-survey), [product-market fit (PMF)](/templates/pmf-survey) surveys, and more.
+
+### How does PostHog compare to Eppo?
+
+Both Eppo and PostHog tools support core feature flag and A/B testing components. The biggest difference is their approach to data warehouses:
+
+- Eppo is warehouse-native, meaning that it integrates with your existing data tools like Snowflake or Postgres. 
+
+- PostHog is not warehouse-native, but instead aims to replace your existing data stack by combining product analytics, [pipelines](/docs/cdp), and warehousing into one platform. This eliminates the need for stitching together integrations between third-party tools, and makes it easier for engineers to work with data. 
+
+Besides this, PostHog offers product analytics, session replays, and surveys, whereas Eppo does not. On the other hand, Eppo offers multi-armed bandit testing (called "contextual bandits" in Eppo) and [CUPED analysis](https://matteocourthoud.github.io/post/cuped/), whereas PostHog does not.
+
+
+<ComparisonTable column1="PostHog" column2="Eppo">
+  <ComparisonRow column1={true} column2={false} feature="Self-serve" description="Free to try, no mandatory sales calls" />
+  <ComparisonRow column1={false} column2={true} feature="Warehouse-native" description="Integrate with your existing data tools like Snowflake or Postgres" />
+  <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Deploy features safely with targeting and percentage rollouts" />
+  <ComparisonRow column1={true} column2={false} feature="Local evaluation" description="Use local, cached flag values to increase speed" />
+  <ComparisonRow column1={true} column2={true} feature="Scheduling" description="Schedule flag updates and rollouts" />
+  <ComparisonRow column1={true} column2={true} feature="Payloads" description="Flags with string, number, or JSON payloads" />
+  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Run tests and see the impact of changes with custom goals and reports" />
+  <ComparisonRow column1={true} column2={true} feature="Multivariate (A/B/n) testing" description="Test multiple variants of a change" />
+  <ComparisonRow column1={true} column2={true} feature="Secondary metrics" description="Monitor impact on unrelated metrics" />
+  <ComparisonRow column1={false} column2={true} feature="Multi-armed bandit" description="Optimize tests automatically by allocating traffic to the best performing variant." /> 
+  <ComparisonRow column1="Bayesian" column2="Bayesian, Frequentist" feature="Statistics engine" description="How the results of an experiment are calculated" />
+  <ComparisonRow column1={true} column2={true} feature="AI/LLM support" description="Compare models with experiments, view performance, cost, and latency" />
```

Comment:
Eppo have done this annoying thing where they call out [supporting AI apps](https://www.geteppo.com/artificial-intelligence) as a core feature, but in reality its not a standalone feature. Instead, its integrating their existing a/b tests code in LLMs. 

Because they call it out, I've added it as a row here. Would love to know if you feel Im potraying us/them accurately

Reply:
I think keep it in.